### PR TITLE
Fix hover underline of icons in name of Tile 

### PR DESCRIPTION
**Pull request by ahackettms**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ npm run change`

#### Description of changes
when the name element of Tile contains SignalField and the Tile is invokeSelection, the whole name field becomes a link and has hover underline. This doesn't look right for the signal icon. 

need to fix in SignalField to have a way to render the text inside as a link or not, so that the underline can be only added to the inner text of SignalField

#### Focus areas to test

(optional)

### CODE REVIEW

File: packages/experiments/src/components/Tile/examples/Tile.Document.Example.tsx [hunk 1]
```diff
@@ -41,17 +41,23 @@ interface IDocumentTileWithThumbnailProps {
...
   const tile = (
     <Tile
       contentSize={{
         width: 176,
         height: 171
       }}
-      itemName={<SignalField before={<TrendingSignal />}>{props.item.name}</SignalField>}
+      itemName={
+        <SignalField before={<TrendingSignal />}>
+          {invokeSelection ? <a>{props.item.name}</a> : <span>{props.item.name}</span>}
```
Comment (ThomasMichon) [inline packages/experiments/src/components/Tile/examples/Tile.Document.Example.tsx]: We don't want an `<a>` element with no semantic purpose.

File: packages/experiments/src/components/Tile/examples/Tile.Document.Example.tsx [hunk 2]
```diff
@@ -41,17 +41,23 @@ interface IDocumentTileWithThumbnailProps {
 const DocumentTileWithThumbnail: React.StatelessComponent<IDocumentTileWithThumbnailProps> = (
   props: IDocumentTileWithThumbnailProps
 ): JSX.Element => {
+  const invokeSelection = true;
```
Comment (ThomasMichon) [inline packages/experiments/src/components/Tile/examples/Tile.Document.Example.tsx]: Why the condition if this is always `true`?

**Comment by dzearing** [comment]
No change file...

**Comment by ahackettms** [comment]
close this for now as will need to try other approach to fix this

**Pull request closed without merging**